Sources/MongoSwift/MongoClient.swift

```diff
@@ -655,6 +655,20 @@ public class MongoClient {
         return self.operationExecutor.execute(operation, client: self, session: session)
     }
 
+    /**
+     * Returns an `EventLoopBoundMongoClient`, a wrapper around this `MongoClient` that will use only the specified
```

Comment:
Do we want to say 'wrapper' in the docstrings?

Reply:
I think its fine to say wrapper. It indicates that the thing returned from this method should have the same behavior as a regular client.